Training/Program.cs

```diff
@@ -1,7 +1,14 @@
-namespace Training {
-    internal class Program {
-        static void Main (string[] args) {
-            Console.WriteLine ("Hello, World!");
-        }
-    }
+// Isogram checker  
+Console.Write ("Enter a string: ");                                       // Gets an input string 
+string isostr = Console.ReadLine ();
+static bool IsIsogram (string isostr) {                                   // A method which checks whether the input is isogram or not
+   char[] result = isostr.ToCharArray ();                                 // Spliting the string into single elements 
+   Array.Sort (result);                                                    // Sorting the elements in ascending order 
+   for (int i = 0; i < isostr.Length - 1; i++) {
+      if (result[i] == result[i + 1])                                     // Checking whether same letter is occured consecutively

```

Comment:
Is it enough if you just check 2 adjacent letters?

Reply:
Yes ravi. Because, we sort the characters in ascending order in the previous step. By doing this the elements will line up in ascending order. After that, it would be sufficient to check the adjacent letters. For example,
Lets take a word "Shresht". When we sort the word in ascending order it becomes like e,h,h,r,s,s,t. So when u check e and h here, it is not equal. So, it will go to next pair, (i.e.,) h and h. Since they are equal, same letter is used twice in the letter. Hence, it is not an isogram. Another example, Lets take the word "isogram". If we sort this using the steps that i used, it becomes a,g,i,m,o,r,s. Here no adjacent letters are equal. Thus, it is an isogram.